src/test/useExplain.coverage.test.js

```diff
@@ -20,63 +20,67 @@ describe('useExplain hook', () => {
     const view = renderHook(useExplain)
     expect(view.result.current.explain).toHaveLength(0)
     act(() => {
-      explainDataManager.setExplain(data.extensions.explain.profiler.data)
+      explainDataManager.setExplain(
+        simplifiedData.extensions.explain.profiler.data
+      )
     })
     view.rerender()
     expect(view.result.current.explain).not.toHaveLength(0)
     expect(view.result.current.explain).toEqual([
       {
+        begin: 1,
+        end: 2,
         path: 'users',
-        begin: 2180524057777125,
-        end: 2180524359067041,
-        time: 301289916,
-        totalBegin: 2180524057777125,
-        totalEnd: 2180524560907916,
-        totalTime: 503130791
+        time: 1,
+        totalBegin: 1,
+        totalEnd: 4,
+        totalTime: 3
       },
       {
+        begin: 2,
+        end: 4,
         path: 'users.0.addresses',
-        begin: 2180524359216958,
-        end: 2180524480481083,
-        time: 121264125,
-        totalBegin: 2180524359216958,
-        totalEnd: 2180524480481083,
-        totalTime: 121264125
+        time: 2,
+        totalBegin: 2,
+        totalEnd: 4,
+        totalTime: 2
       },
       {
+        begin: 3,
```

Comment:
This value (and other values in this structure) is not correct. The begin of the `users.0` should be abter the end of the parent `user`.